draft-ietf-rtcweb-jsep.xml

```diff
@@ -1932,6 +1934,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             be added, as specified in
             <xref target="I-D.ietf-ice-trickle"></xref>, Section 4.</t>
 
+            <t>An "a=ice-options" line with the "ice2" option MUST
```

Comment:
ice2 should be combined into the previous paragraph that discusses ice-options.

Reply:
Whoops, duh. Fixed.